### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by natchengithub** [community/community#125741] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://wordpress.org/support/topic/seems-not-working-with-wordpress-6-0/
If you can update plugin The Paste to the newer versions of WordPress, please, to make it works!!! This plugin saves my life!!!

**Comment by Abhishek22K** [comment]
The issue described in the forum thread is that the plugin "The Paste" is not working with WordPress versions 6.0 and above. Users report that the plugin, although activated, does not appear in the media library. This issue persists even with the latest updates and no other active plugins. One user noted that it still works in the original content field but not in custom fields.

To resolve this, you can:

Contact the plugin developer for an update.
Look for alternative plugins that offer similar functionality.
Manually update the plugin if you're comfortable with coding.

**Comment by LiteBrite82** [comment]
Hi @natchengithub ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/mcguffin/the-paste). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

P.S. Also consider changing the title of your discussion to reflect the issue you are having; for better community support.
